.github/actions/llm/cli-test/action.yml

```diff
@@ -1,12 +1,13 @@
 name: 'llm-cli Flow Verification (Linux)'
 description: 'Verify the llm-cli flow on linux'
+
 runs:
   using: "composite"
   steps:
     - name: Test llama llm-cli
       shell: bash
       run: |
-        source $CONDA_HOME/bin/activate bigdl-init-llm
+        source $CONDA_HOME/bin/activate bigdl-llm-test
         $CONDA_HOME/bin/conda info
         
         llm-cli -t $THREAD_NUM -n 256 -x llama -m $LLAMA_INT4_CKPT_PATH -p 'Once upon a time,'
```

Comment:
Maybe we should check the outputs in the following PR.

Reply:
Yes, we should check the perplexity of test outputs later.